lib/src/editor/editor_component/service/shortcuts/command_shortcut_events/toggle_colors_command.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+/// Toggle Color Commands
+///
+/// Cmd / Ctrl + Shift + H: toggle highlight color
+/// Cmd / Ctrl + Shift + T: toggle text color
+/// - support
+///   - desktop
+///   - web
+
+List<CommandShortcutEvent> toggleColorCommands({
+  ToggleColorsStyle? style,
+}) =>
+    [
+      toggleHighlightCommand(
+        style: style ?? ToggleColorsStyle(),
+      ),
+    ];
+
+class ToggleColorsStyle {
+  ToggleColorsStyle({
+    this.highlightColor = const Color(0x60FFCE00),
+  });
+
+  final Color highlightColor;
+}
+
+CommandShortcutEvent toggleHighlightCommand({
+  required ToggleColorsStyle style,
+}) =>
+    CommandShortcutEvent(
+      key: 'toggle highlight',
+      command: 'ctrl+shift+h',
+      macOSCommand: 'cmd+shift+h',
+      handler: (editorState) => _toggleHighlight(editorState, style: style),
+    );
+
+KeyEventResult _toggleHighlight(
+  EditorState editorState, {
+  required ToggleColorsStyle style,
+}) {
+  if (PlatformExtension.isMobile) {
+    assert(false, 'toggle highlight is not supported on mobile platform.');
+    return KeyEventResult.ignored;
+  }
+
+  final selection = editorState.selection;
+  if (selection == null) {
```

Comment:
```suggestion
  if (selection == null || selection!.isCollapsed) {
```
We don't need to do anything if it's collapsed right?